deploy/init-db/init-db.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
For reviewers: This file is passed in to the postgres container when running locally, and used to initialize databases, users, and tables.

UCP and Applications-RP will each have separate databases. We **DO NOT** want cross-component database reads.

Reply:
I understand not wanting to have cross-component database reads.
But today, we have it this way, correct? 
Does this mean going forward, UCP datastore will contain data like planes where are only the ones that are handled by RP will be stored in Applications-RP's instance of db? Also, do scope queries work with postgres identical to etcd?

Reply:
> But today, we have it this way, correct?

Today it's possible but discouraged. There's nothing stopping us except code review. 

> Does this mean going forward, UCP datastore will contain data like planes and ucp resource-groups where as only the ones that are handled by RP will be stored in Applications-RP's instance of db? 

Correct. 

Keep in mind it's a single database server, with separate usernames and databases within the server. 

> Also, do scope queries work with postgres identical to etcd?

Yes. The tests are shared between all of the datastore implementations.